### PR TITLE
util-linux: update to 2.35.2.

### PR DESCRIPTION
**Pull request by ailiop-git**

### CODE REVIEW

**Comment by Hoshpak** [comment]
fails to build with musl.

**Comment by ailiop-git** [comment]
> fails to build with musl.

yeah it's due to our old musl, we're missing a45289112c033fe that defines pidfd_open, which can be easily backported (applies cleanly), not sure though if this is sufficient (I think the musl cross-compilers would also need to be bumped).

**Comment by ericonr** [comment]
@ailiop-git if you want to mess with musl, you could look into applying these, perhaps: https://www.openwall.com/lists/musl/2020/05/22/10

We might need some discussion on IRC on how best to apply them, though.

File: srcpkgs/util-linux/template [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Template file for 'util-linux'
 # Keep this package sync with util-linux-libs
 pkgname=util-linux
-version=2.35.1
```
Comment (sgn) [inline srcpkgs/util-linux/template:4]: There is a note in this file:

    # Keep this package sync with util-linux-libs

**Comment by sgn** [comment]
Superseded by #23005

**Pull request closed without merging**